yateto/gemm_configuration.py

```diff
@@ -159,10 +167,18 @@ def __init__(self, arch, threshold: int = 128):
     self._threshold = threshold
 
   def _archSupported(self):
-    return self._arch.name.lower() in {'thunderx2t99', 'knl', 'skx'}
+    supported_set = {'thunderx2t99', 'knl', 'skx'}
+    if self._arch.name.lower() in supported_set:
+      return True
+    else:
+      if self._arch.host_name and self._arch.host_name.lower() in supported_set:
+        return True
+      else:
+        return False
```

Comment:
Should be return self._arch.host_name and self._arch.host_name.lower() in supported_set